[WPE][Qt6] qt-wpe-mini-browser close button and url onAccepted event

Qingwu-Li opened the pull request:
#### 39da07ebbe42e33bca448ee37ca2799616bbb751
<pre>
[WPE][Qt6] qt-wpe-mini-browser close button and url onAccepted event

<a href="https://bugs.webkit.org/show_bug.cgi?id=281953">https://bugs.webkit.org/show_bug.cgi?id=281953</a>

Reviewed by Nikolas Zimmermann.

Add a close button to exit the application.
URL loading on Return key press in the address bar.

Signed-off-by: LI Qingwu &lt;Qing-wu.Li@leica-geosystems.com.cn&gt;
Canonical link: <a href="https://commits.webkit.org/285605@main">https://commits.webkit.org/285605@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ff4f424d8b0c5411bb2edf6d488dedaac8d2dbbb

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/73192 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/52621 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/26000 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/77418 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/24430 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/75307 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/60426 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/403 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/57517 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/15990 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/76259 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/47537 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/62987 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/37935 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/44176 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/20465 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/22759 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/66029 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/20819 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/79074 "Built successfully") | 
| | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-2-Build-EWS "Waiting in queue, processing has not started yet") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/81 "Build is in progress. Recent messages:OS: Sequoia (15.0.1), Xcode: 16.0; Skipping applying patch since patch_id isn't provided; Checked out pull request; Skipped layout-tests; Running layout-tests; Running upload-test-results") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/65947 "Passed tests") | 
| | [  ~~🛠 vision-sim~~](https://ews-build.webkit.org/#/builders/121/builds/648 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/62996 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/65230 "Passed tests") | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-2-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/7228 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/471 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/500 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/485 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/502 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/413acfe5f44ee5ed6c10d34a2d29abf203327722. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/35347

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/b45fc86e002d4c8b7f9b25e4759ba773ae7a9af0. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/35347

Qingwu-Li commented:
Gentle ping...

mcatanzaro commented:
Hi, can you please check the [commit message guidelines](https://webkit.org/contributing-code/#commit-messages)? In particular, the line "Reviewed by NOBODY (OOPS!)" is mandatory.

A couple other things:

 * Please add the prefix "[WPE]" to the start of your commit message, to disambiguate which MiniBrowser you're working on
 * Please report a bug on WebKit Bugzilla corresponding to this pull request. The title of your bug report should match the first line of your commit message here. Yes, creating a bug report for every pull request is a waste of time, but it's expected.

nikolaszimmermann reviewed (approved):
Thanks, looks useful!

Qingwu-Li committed:
[WPE][Qt6] qt-wpe-mini-browser close button and url onAccepted event

https://bugs.webkit.org/show_bug.cgi?id=281953

Reviewed by Nikolas Zimmermann.

Add a close button to exit the application.
URL loading on Return key press in the address bar.

Signed-off-by: LI Qingwu <Qing-wu.Li@leica-geosystems.com.cn>
Canonical link: https://commits.webkit.org/285605@main

webkit-commit-queue commented:
Committed 285605@main (39da07ebbe42): <https://commits.webkit.org/285605@main>

Reviewed commits have been landed. Closing PR #35347 and removing active labels.
